v2/internal/wv2installer/manual.go

```diff
@@ -0,0 +1,11 @@
+//go:build wv2runtime.manual
+// +build wv2runtime.manual
+
+package wv2installer
+
+import "github.com/wailsapp/wails/v2/pkg/options/windows"
+
+func doInstallationStrategy(installStatus installationStatus, messages *windows.Messages) error {
+	// fallback for manually specifying webview2
+	return nil
```

Comment:
During the implementation of PR #1456 I took a closer look into the installing process and now I'm unsure if we really need this manual strategy, or at least we should never return `nil` here.

Currently the `doInstallationStrategy` is called if no webview could be found or if the webview is outdated (does not meet the minimum version requirement). In the fixed version case, `no webview` (e.g. wrong folder) will be catched during the `chromium.Embed`. If the webview is outdated and by returning nil here, the creation of the webview will succeed but later on it could lead to strange panics like `AccessViolationException` because COM interfaces are used that aren't part of the outdated version.
On the other hand having a fixed version with any strategy other than `error` might be confusing. If e.g. the fixed version is outdated and the default strategy is used, the user will see a download confirmation dialog. But that will just install/update the system wide webview and it will never fix the problem.

What would you think about the following @leaanthony and @NanoNik?
 > If a fixed version is used and it would lead to a `doInstallationStrategy` call, then just error with an appropriate message similar to the `error` strategy. It won't matter which strategy has been selected during the build, it will just be ignored.

Reply:
Given that insight, perhaps we could get some more context on the problem this PR is trying to solve.

Reply:
I think the PR is perfectly fine and it's a great feature to allow to distribute the Webview2 runtime with the app itself and to only use that runtime.

My comment was referring to if we need another Webview2 installer strategy in `wails build`:
```
 -webview2 string
    	WebView2 installer strategy: download,embed,browser,error. (default "download")
```

Reply:
Maybe we could just move the implementation of the error strategy into the common part?
Then the error strategy could simply be used as a trampoline to that and we have the implementatiom available independent from the build tags.